### PR TITLE
RefCell::borrow_mut example should demonstrate mut

### PR DESCRIPTION
**Pull request by frewsxcv**

### CODE REVIEW

File: src/libcore/cell.rs [hunk 1]
```diff
@@ -414,7 +414,12 @@ impl<T: ?Sized> RefCell<T> {
     ///
     /// let c = RefCell::new(5);
     ///
-    /// let borrowed_five = c.borrow_mut();
+    /// {
+    ///     let mut borrowed_cell = c.borrow_mut();
+    ///     *borrowed_cell = 7;
+    /// }
```
Comment (alexcrichton) [inline src/libcore/cell.rs]: Perhaps this could just be:

```rust
*c.borrow_mut() = 7;
```

Comment (frewsxcv) [inline src/libcore/cell.rs]: Done.

**Pull request merged**